.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "libstorj"]
+	path = lib/libstorj
+	url = https://github.com/bryanchriswhite/libstorj.git
```

Comment:
are we planning to have this submodule in master ultimately?

Reply:
Good question; I don't know. I like having it as a submodule and I think this is getting removed again at some point.

Reply:
after many long battles with submodules, i'm not opposed to them, but i think i'd rather not see submodules in our go v3 repo. can we have the new libstorj submodule in storj/storj instead of vice versa?

e.g., storj/libstorj has the submodule, instead of this